LayoutTests/fast/scrolling/mac/smooth-scroll-recursive-frame-to-frame.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html> <!-- webkit-test-runner [ useFlexibleViewport=true EventHandlerDrivenSmoothKeyboardScrollingEnabled=true ] -->
```

Comment:
`useFlexibleViewport=true` doesn't do anything on macOS, and these are macOS tests so you can remove it.